src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleProductsController.cs

```diff
@@ -288,7 +288,8 @@ public async Task<ActionResult> DeleteProduct([FromQuery] string[] ids)
         /// </summary>
         /// <returns></returns>
         [HttpPost]
-        [Route("associations/search")]
+        [Route("associations/search")]
+        [Obsolete("use POST api/catalog/products/{productid}/associations/search instead")]
```

Comment:
Will need to change this route 'api/catalog/products/{productid}/associations/search' to the changed search one.